src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -91,11 +92,17 @@ public JurisdictionDefinition getJurisdiction(String jurisdictionId) {
         return caseDefinitionRepository.getJurisdiction(jurisdictionId);
     }
 
+    @Override
+    public Optional<List<String>> getAllCaseTypesByJurisdictions(List<String> jurisdictionIds) {
```

Comment:
No need to use Optional with lists. Just have it return an empty list in case of no results